src/main/java/uk/gov/beis/els/categories/televisions/model/TelevisionsForm.java

```diff
@@ -4,39 +4,105 @@
   @DualModeField
   private String efficiencyRating;
 
   @FieldPrompt("Is there a visible power switch which puts the television in a condition with power consumption not exceeding 0.01 Watts?")
-  @NotNull(message = "Specify if there is a visible power switch")
+  @NotNull(message = "Specify if there is a visible power switch", groups = PreMarch2021Field.class)
   private Boolean powerSwitch;
 
   @FieldPrompt("On-mode power consumption in Watts")
-  @Digits(integer = 3, fraction = 0, message = "Enter the power consumption, up to 3 digits long")
+  @Digits(integer = 3, fraction = 0, message = "Enter the power consumption, up to 3 digits long", groups = PreMarch2021Field.class)
   private String powerConsumption;
 
   @FieldPrompt("Annual on-mode energy consumption [kWh/annum]")
-  @Digits(integer = 3, fraction = 0, message = "Enter the annual energy consumption, up to 3 digits long")
+  @Digits(integer = 3, fraction = 0, message = "Enter the annual energy consumption, up to 3 digits long", groups = PreMarch2021Field.class)
   private String annualEnergyConsumption;
 
-  @FieldPrompt("Visible diagonal screen size in centimetres")
-  @Digits(integer = 3, fraction = 0, message = "Enter the screen size in centimetres, up to 3 digits long")
-  private String screenSizeCm;
+  // Post march 2021 fields
+  @FieldPrompt("Energy efficiency class for SDR content")
+  @NotBlank(message = "Select an energy efficiency class for SDR content", groups = PostMarch2021Field.class)
+  @DualModeField
+  private String efficiencyRatingSdr;
 
-  @FieldPrompt("Visible diagonal screen size in inches")
-  @Digits(integer = 3, fraction = 0, message = "Enter the screen size in inches, up to 3 digits long")
-  private String screenSizeInch;
+  @FieldPrompt("On-mode energy consumption in kWh per 1000 hours, when playing SDR content")
+  @Digits(integer = 3, fraction = 0, message = "Enter the energy consumption when playing SDR content, up to 3 digits long", groups = PostMarch2021Field.class)
+  private String energyConsumptionSdr1000h;
+
+  @FieldPrompt("Can this product display HDR content?")
+  @NotNull(message = "Select whether this model can display HDR content", groups = PostMarch2021Field.class)
+  private Boolean isHdr;
+
+  @FieldPrompt("Energy efficiency class for HDR content")
+  @NotBlank(message = "Select an energy efficiency class for HDR content", groups = HdrGroup.class)
+  private String efficiencyRatingHdr;
+
+  @FieldPrompt("On-mode energy consumption in kWh per 1000 hours, when playing HDR content")
+  @Digits(integer = 3, fraction = 0, message = "Enter the energy consumption when playing HDR content, up to 3 digits long", groups = HdrGroup.class)
+  private String energyConsumptionHdr1000h;
+
+  @FieldPrompt("Horizontal resolution in pixels")
+  @Digits(integer = 4, fraction = 0, message = "Enter the horizontal resolution in pixels, up to 4 digits long", groups = PostMarch2021Field.class)
```

Comment:
16k displays are now a thing so perhaps we should allow up to 5 digits horizontal and vertical: https://www.bbc.co.uk/news/technology-47867038

That model is over 10m on the diagonal too, so maybe we should allow up to 4 digits in the cm field

Reply:
Have updated, tested both the old and new labels (since screen size is on both) and they look fine with the increased limits.